src/server/api/schema.js

```diff
@@ -3482,6 +3482,29 @@ const rootResolvers = {
         organization_id: parseInt(organizationId, 10)
       });
       return formatPage(query, { after, first });
+    },
+    campaignNavigation: async (
+      _root,
+      { campaignId, organizationId },
+      { user }
+    ) => {
+      await accessRequired(user, organizationId, "SUPERVOLUNTEER");
+
+      const {
+        rows: [{ prevcampaign }]
+      } = await r.knex.raw(
+        `SELECT max(id) as prevCampaign FROM campaign where id < ${campaignId} AND is_started = TRUE AND organization_id = ${organizationId} AND is_archived = FALSE;`
```

Comment:
Since these buttons will be visible on archived campaigns, it may make sense to check is_archived for the current campaign and select prevCampaign where is_archived matches. It may be confusing if the user clicks the button on an archived campaign page, and we jump to an unarchived campaign. Defer to you if there is a better solution here though!